02Lesson/02Project/index.html

```diff
@@ -1,10 +1,52 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css"/>
-</head>
-<body>
-  <!-- You code start here -->
-</body>
+  <head>
+    <title>
+      The Daily Planet
+    </title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/style.css" type="text/css"/>
+  </head>
+  <body>
+    <h1>
+      Daily Planet!!
+    </h1>
+    <nav>
+      <a href="http://www.espn.com/">Sports</a> |
+      <a href="http://www.statesman.com/news/local/">Local</a> |

```

Comment:
Don't leave stray text floating around unless it's contained in an HTML element (I'm referring to the `|`)